src/plugins/displayLayout/components/DisplayLayout.vue

```diff
@@ -246,7 +248,7 @@
                 } else {
                     let identifier = this.openmct.objects.makeKeyString(domainObject.identifier);
 
-                    if (!this.objectViewMap[identifier]) {
+                    if (!this.containsObject(identifier)) {
```

Comment:
@akhenry This PR got merged before I had a chance to review it. As discussed, objectViewMap tracks the objects internally and is a less expensive check than containsObject(). A similar check is done in addChild() and removeChild() and would be good to keep it consistent with those.